src/pip/_internal/commands/__init__.py

```diff
@@ -59,6 +59,10 @@
         'pip._internal.commands.cache', 'CacheCommand',
         "Inspect and manage pip's wheel cache.",
     )),
+    ('index', CommandInfo(
+        'pip._internal.commands.index', 'IndexCommand',
+        "Inspect information available from package indexes.",
```

Comment:
Hmm, thi uses *indexes* but the news fragment uses *indices*, we probably want to unify them. Are there existing usages in the code base?

Reply:
most of the places use `indexes`, I will update the news section

Reply:
We've pretty much always used indexes, mostly because I like that spelling and now the codebase is full of that. :P